src/view/WorkspaceManager.js

```diff
@@ -191,6 +201,7 @@ define(function (require, exports, module) {
      */
     Panel.prototype.hide = function () {
         Resizer.hide(this.$panel[0]);
+        exports.trigger(EVENT_WORKSPACE_PANEL_HIDDEN, this.panelID);
```

Comment:
Instead of having two events `EVENT_WORKSPACE_PANEL_HIDDEN` and `EVENT_WORKSPACE_PANEL_SHOWN`, can we have just one event and pass in the param whether the panel is hidden or shown?